umh.docs.umh.app/content/en/docs/production-guide/backup_recovery/backup-restore-umh.md

```diff
@@ -73,6 +84,33 @@ sudo $(which kubectl) scale statefulset {{< resource type="statefulset" name="ka
 sudo $(which kubectl) scale statefulset {{< resource type="statefulset" name="mqttbroker" >}} --replicas=0 -n united-manufacturing-hub --kubeconfig /etc/rancher/k3s/k3s.yaml
 ```
 
+### Copy kubeconfig file
+
+1. Show the content of kubeconfig file with the following command and copy the output:
```

Comment:
Specify that this command needs to be executed from the instance

Reply:
specified that one should run it in the instance's shell